pom.xml

```
@@ -35,6 +27,161 @@
     </license>
   </licenses>
 
+  <scm>
+    <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
+    <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
+    <tag>${scmTag}</tag>
+    <url>https://github.com/${gitHubRepo}</url>
+  </scm>
+
+  <properties>
+    <revision>6.20</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <jetty.version>10.0.20</jetty.version>
+    <slf4j.version>2.0.13</slf4j.version>
+    <gitHubRepo>jenkinsci/winstone</gitHubRepo>
+    <spotless.check.skip>true</spotless.check.skip>
```

With `spotless.check.skip` set to `true`, I think that will not check that the spotless formatting is being preserved across commits.  Most other pull requests that enable spotless have chosen to set this to `false` so that spotless is checked as part of the typical build process.
```suggestion
    <spotless.check.skip>false</spotless.check.skip>
```

---

sorry just a bad copy paste.
fixed